code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -871,16 +880,29 @@
 				separated = TRUE
 				var/obj/item/dogtag/tag = new tag_type(get_turf(user))
 				user.put_in_hands(tag)
+				tag.desc = desc
 				separated = TRUE
 				update_icon()
+		if(user.a_intent == I_HELP)
+			if(!tucked)
+				var/mob/living/carbon/human/H = user
+				to_chat(user, SPAN_NOTICE("You tuck \the [src] under your [H.w_uniform.name]."))
+				tucked = !tucked
+				update_icon()
+			else
+				var/mob/living/carbon/human/H = user
```

Comment:
If you're using this var in either cases, you might aswell extract this assignment (and the verification logic) out of the two cases, since you're doing this either way